codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "src/ethereum/*_glacier"  # ignore folders and all its contents
```

Comment:
Could you also include the dao fork? It's essentially a glacier fork and is covered by the same linting tool.